R/zchunk_L133.water.demand.livestock.R

```diff
@@ -36,41 +35,128 @@ module_water_L133.water.demand.livestock_DISABLED <- function(command, ...) {
     L100.FAO_an_Stocks <- get_data(all_data, "L100.FAO_an_Stocks")
     L100.FAO_an_Dairy_Stocks <- get_data(all_data, "L100.FAO_an_Dairy_Stocks")
 
+    # Silence package checks
+    year <- iso <- item <- value <- dairy.to.total <- dairy.adj <-
+      coefficient <- GCAM_region_ID <- GCAM_commodity <- water.consumption <-
+      water_type <- coefficient <- Coefficient <- NULL
+
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
+    # Calculate livestock water coefficients by region ID / GCAM_commodity/ water type.
+
+    # Start by finding  the number of non-dairy producing livestock.
+
+    # Create a tibble of dairy producing stocks by country and FAO animal
+    # product name. Only use stock information from the year 2000 since that is the
+    # year the water use coefficients are from. This tibble will be used in the next step to adjust
+    # the total livestock for the
```

Comment:
incomplete comment. I think it's adjust total livestock to remove dairy animals.